src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -43,6 +43,11 @@ public static int Run(string[] args)
             CommandOption versionSuffixOption = app.Option(
                $"--version-suffix <{LocalizableStrings.VersionSuffixOption}>", LocalizableStrings.VersionSuffixOptionDescription,
                 CommandOptionType.SingleValue);
+
+            CommandOption profileProjOption = app.Option(
+               $"--profile <{LocalizableStrings.ProfileProjOption}>", LocalizableStrings.ProfileProjOptionDescription,
+                CommandOptionType.SingleValue);
```

Comment:
i have a bug filed for it, it is not as simple as passing multiple arguments, we would need to merge it properly